1.assignment.js

```diff
@@ -0,0 +1,13 @@
+
+let a;
+console.log("value of a before assignment is:",a);

```

Comment:
Good work!
Expecting some more comments to describe different cases for you to have a future reference (this is applicable throughout the learning journey)